api/maven-api-model/src/main/mdo/maven.mdo

```diff
@@ -3384,10 +3383,6 @@
           <description><![CDATA[
             For a plugin project (packaging is {@code maven-plugin}), the minimum version of
             Maven required to use the resulting plugin.<br>
```

Comment:
We can remove `<br>` if there is no longer another paragraph after it.

Reply:
done